packages/notepadplusplus.vm/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,32 @@
+$ErrorActionPreference = 'Stop'
```

Comment:
File is a duplicate and has an uppercase letter. I wonder if we can have GH hook to check for lowercase file paths?

Reply:
We could. Although this is something easy to fix if we miss it. I would like to first focus on automatic updates or fixes.